docker-compose.yml

```diff
@@ -1,39 +1,19 @@
-# This is the docker-compose file for the dev environment.
-# It can be used to quickly run 4 services:
-#   1) The front-end web server
-#   2) The back-end API server
-#   3) The Postgres database
-#   4) The Redis cache
-# You can start all services by running "docker-compose up"
+# This is the docker-compose file to run Medplum's required background serivces
+# It can be used to quickly run 2 services:
+#   1) The Postgres database
+#   2) The Redis cache
+# You can start all services by running "docker-compose -f docker-compose-background.yml up"
```

Comment:
Should this just be `docker-compose up` ?

Reply:
yes